package_manifest.yaml

```diff
@@ -1111,3 +1101,66 @@ pulpcore_packages:
     python-whitenoise: {}
     python-yarl: {}
     python-zipp: {}
+
+foreman_release_packages:
+  vars:
+    package_base_dir: 'packages/foreman/'
+    repoclosure_config: repoclosure/yum.conf
+    repoclosure_target_repos:
+      el8:
+        - el8-foreman-client-nightly
+      rhel7:
+        - el7-foreman-nightly
+        - el7-foreman-client-nightly
+      rhel6:
+        - el6-foreman-client-nightly
+      rhel5:
+        - el5-foreman-client-nightly
+      fedora29:
+        - f29-foreman-client-nightly
+    repoclosure_lookaside_repos:
+      el8:
+        - el8-baseos
+        - el8-appstream
+        - el8-epel
+        - el8-extras
+        - el8-puppet-6
+      rhel7:
+        - el7-base
+        - el7-updates
+        - el7-epel
+        - el7-extras
+        - el7-puppet-6
+      rhel6:
+        - el6-base
+        - el6-updates
+        - el6-epel
+        - el6-extras
+        - el6-puppet-6
+        - el6-qpid
+        - el6-subscription-manager
+        - el6-pulp
+      rhel5:
+        - el5-base
+        - el5-updates
+        - el5-epel
+        - el5-extras
+        - el5-puppet-6
+        - el5-subscription-manager
+        - el5-pulp
+      fedora29:
+        - f29-base
+        - f29-updates
+        - f29-puppet-6
+  hosts:
+    foreman-release:
+      diff_package_tags:
+        - foreman-nightly-nonscl-rhel7
+        - foreman-client-nightly-el8
+        - foreman-client-nightly-rhel7
+        - foreman-client-nightly-rhel6
+        - foreman-client-nightly-rhel5
+        - foreman-client-nightly-fedora29
+      releasers:
+        - koji-foreman
+        - koji-foreman-client
```

Comment:
diff_package_tags and releasers could be moved to group vars at this point, but I won't block on this

Reply:
I went ahead and moved em.